examples/rosettacode/24_game.nit

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: 24 game
+# SEE: <http://rosettacode.org/wiki/24_game>
+
+redef class Char 
+	fun is_op: Bool do return "-+/*".has(self)
+	fun is_dig: Bool do return [0..9].has(self.to_i)
```

Comment:
Is there a fundamental difference with [`Char::is_digit`](http://nitlanguage.org/doc/stdlib/class_standard__kernel__Char.html#article:standard__kernel__Char__is_digit.definition)?